sql_test.py

```diff
@@ -0,0 +1,40 @@
+import mysql.connector
+from mysql.connector import errorcode
+
+# Obtain connection string information from the portal
+config = {
+    'host':'mictestdb.mysql.database.azure.com',
+    'user':'c_hawk7397',
+    'password':'m1(|<3yM0i_i53',
+    'database':'mictestdb',
+    'client_flags': [mysql.connector.ClientFlag.SSL],
+    'ssl_ca': 'DigiCertGlobalRootCA.crt.pem'
+}
+
+# Construct connection string
+try:
+    conn = mysql.connector.connect(**config)
+    print("Connection established")
+except mysql.connector.Error as err:
+    if err.errno == errorcode.ER_ACCESS_DENIED_ERROR:
+        print("Something is wrong with the user name or password")
+    elif err.errno == errorcode.ER_BAD_DB_ERROR:
+        print("Database does not exist")
+    else:
+        print(err)
+else:
+    cursor = conn.cursor()
+
+    # Read data
+    cursor.execute("SELECT * FROM GlossaryEntry;")
+    rows = cursor.fetchall()
```

Comment:
We should note that if the database were to grow substantially beyond the current expected size, this would be something that could be altered. Trying to retrieve the whole thing at once would consume a lot of memory, so iteration may be a worthwhile solution for future teams to explore if they develop this app further.

Reply:
This was just for testing a connection, in actual implementation I don't think we would need to retrieve the existing db contents.